src/components/Slider.tsx

```diff
@@ -146,42 +52,89 @@ const Slider: React.FC<IProps> = ({ children, onSlideChange }) => {
   }, [embla, setScrollSnaps, onSelect])
 
   return (
-    <Embla>
-      <EmblaViewport ref={emblaRef}>
-        <EmblaContainer>{children}</EmblaContainer>
-      </EmblaViewport>
-      <Buttons>
-        <Button onClick={scrollPrev} disabled={!prevBtnEnabled}>
-          <Icon
-            name="arrowLeftIos"
-            color={
-              prevBtnEnabled
-                ? theme.colors.slider.btnColor
-                : theme.colors.slider.btnColorDisabled
-            }
-          />
-        </Button>
-        <Button onClick={scrollNext} disabled={!nextBtnEnabled}>
-          <Icon
-            name="arrowRightIos"
-            color={
-              nextBtnEnabled
-                ? theme.colors.slider.btnColor
-                : theme.colors.slider.btnColorDisabled
-            }
-          />
-        </Button>
-      </Buttons>
-      <Dots>
+    <Box
+      position="relative"
+      p="2rem"
+      borderWidth="1px"
+      borderStyle="solid"
+      borderRadius="0.3rem"
```

Comment:
Lets use `base` here (which is `0.25rem`) to be consistent with the rest of the site.

```suggestion
      borderRadius="base"
```